lib/fetch/sanitize.js

```diff
@@ -164,22 +168,46 @@ export function sanitizeBadger(json) {
     lastName: name && name.split(' ').slice(-1).join(' '),
     order: get('badger.order'),
     jobTitle: get('badger.job-title'),
-    primaryImageUrl: primaryImageUrl && primaryImageUrl.replace(/^http:/, 'https:'),
-    secondaryImageUrl: secondaryImageUrl && secondaryImageUrl.replace(/^http:/, 'https:'),
+    primaryImageUrl: primaryImageUrl &&
```

Comment:
sure, prettier's fine. but we shouldn't just turn it on because it edits things that haven't changed - this PR doesn't change badgers but prettier made the edit! you see what i mean?